types/ObservationType.json

```diff
@@ -0,0 +1,106 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "/typesObservationType",
+    "description": "Abstract class that supports any observation or activity on a geospatial feature or crop",
+    "type": "object",
+    "required": [
+        "observationDate"
+    ],
+    "allOf": [
+        {
+            "$ref": "../types/ResourceType.json"
+        },
+        {
```

Comment:
Not sure this "required" clause is quite right.
For an Observation I think we require **observationDate** and **feature**.
- featureId and holdingId are not attributes
- holding will be required in some cases, but not all.
Overall, my recommendation is to NOT have required fields in abstract types, and instead specify them in concrete types.